src/mainloop.rkt

```diff
@@ -321,9 +326,9 @@
   (run-improve! iters prog specification preprocess original-points repr))
 
 
-(define (run-improve! prog pcontext iters
-                      #:specification [specification #f]
-                      #:preprocess [preprocess '()])
+(define (make-preprocess-pcontext prog pcontext iters
+                                  #:specification [specification #f]
+                                  #:preprocess [preprocess '()])
```

Comment:
Not sure why this was done but it's fine. This part of mainloop is kind of ugly anyway.